lib/fathead/numpy/parse.py

```diff
@@ -206,16 +213,52 @@ def get_abstract(section_div):
         rendered_returns = RETURNS_TEMPLATE.format(returns=returns)
     else:
         rendered_returns = ""
+    
+    examples = get_examples(section_div, "rubric")
+    if examples:
+        rendered_examples = EXAMPLES_TEMPLATE.format(examples=examples)
+    else:
+        rendered_examples = ""    
 
     abstract = ABSTRACT_TEMPLATE.format(
         rendered_information=scrub_text(rendered_information),
         rendered_signature=scrub_text(rendered_signature),
         rendered_parameters=scrub_text(rendered_parameters),
-        rendered_returns=scrub_text(rendered_returns)
+        rendered_returns=scrub_text(rendered_returns),
+        rendered_examples=scrub_text(rendered_examples)
     )
     return abstract
 
 
+def get_examples(section_div, examples_class):
+    """Parse and return the parameters or returns of the documentation topic.
+
+    Parameters
+    ----------
+    section_div : bs4.BeautifulSoup
+        The BeautifulSoup object corresponding to the div with the "class"
+        attribute equal to "section" in the html doc file.
+
+    examples_class: Str
+        The value of the "class" attribute of the <p> tag within section_div.
+        If "rubric" is present, then only the examples for the topic is there.
+
+    Returns
+    -------
+    Str:
+        The examples for the topic.
+    """
+    examples_present = section_div.find("p", attrs={"class": examples_class})
+    if not examples_present:
+        return
+    examples = section_div.find("pre")
+    examples = str(examples)
+    if examples.endswith('</pre>') and examples.startswith('<pre>'):
+        examples = examples[:-6]
+        examples = examples[5:]
```

Comment:
This isn't the right way to extract text from within the `pre` tags. 

1. This python slicing (or the earlier conditional) wont work if there's whitespace before the `<pre>` or after `</pre>`

2. You're just removing the `pre` tags. There is still a lot of html markup inside the `pre` tags that is unnecessarily included in the output. 

E.g. For "numpy arcos" this is output for the examples section:
```
<span class="gp">&gt;&gt;&gt; </span><span class="n">np</span><span class="o">.</span><span class="n">arccos</span><span class="p">([</span><span class="mi">1</span><span class="p">,</span> <span class="o">-</span><span class="mi">1</span><span class="p">])</span>
<span class="go">array([ 0.        ,  3.14159265])</span>
```
These extra spans and classes should be removed. 


The easier and cleaner way to do this would be use `examples.text` (or `examples.string` or `examples.contents`) to extract the actual text within those tags. Checkout the beautifulsoup4 documentation.